src/Symfony/Component/Serializer/SerializerInterface.php

```diff
@@ -19,14 +19,20 @@ interface SerializerInterface
     /**
      * Serializes data in the appropriate format.
      *
-     * @param mixed  $data    Any data
-     * @param string $format  Format name
-     * @param array  $context Options normalizers/encoders have access to
```

Comment:
I think this description can be useful for developers, isn't it?

Reply:
It's kept for $context. The others are useless to me also.

Reply:
Sorry. Missed the next line.

I agree with you.